common/scala/src/main/resources/application.conf

```diff
@@ -1,2 +1,12 @@
 # default application configuration file for spray/akka
-include "logging"
\ No newline at end of file
+include "logging"
+
+akka.http {
+    client {
+        parsing.illegal-header-warnings = off
```

Comment:
This gets rid of those annoying warnings that we get from CouchDB responses, btw.